packages/@o3r/eslint-plugin/src/rules/json/json-dependency-versions-harmonize/json-dependency-versions-harmonize.spec.ts

```diff
@@ -1,25 +1,18 @@
+import * as fs from 'node:fs/promises';
 import * as path from 'node:path';
-import {
-  cleanVirtualFileSystem,
-  useVirtualFileSystem
-} from '@o3r/test-helpers';
 import {
   RuleTester
 } from '@typescript-eslint/rule-tester';
-import jsonParser from 'jsonc-eslint-parser';
-import jsonDependencyVersionsHarmonize from './json-dependency-versions-harmonize';
-
-const virtualFileSystem = useVirtualFileSystem();
+import * as jsonParser from 'jsonc-eslint-parser';
+import jsonDependencyVersionsHarmonize, {
+  VersionsHarmonizeOptions
+} from './json-dependency-versions-harmonize';
 
 const ruleTester = new RuleTester({
   languageOptions: {
-    parser: jsonParser,
-    parserOptions: {
-      ecmaVersion: 2018,
-      sourceType: 'module'
-    }
+    parser: jsonParser
   }
-});
+} as any);
```

Comment:
I don't understand why the any is needed here... but without it's failing because it says that `languageOptions` does not exist on `FlatConfig.Config` (which is false, it does)